driver/src/main/scala/api/commands/writeresult.scala

```diff
@@ -32,8 +32,7 @@ sealed trait WriteResult {
 object WriteResult {
   def lastError(result: WriteResult): Option[LastError] = result match {
```

Comment:
yes, with this change, I get a failed future with a `LastError` instead of `GenericDriverException`.

To get a `MultiBulkWriteResult`(in a successful future) one have to use `ordered = false`.
I think this was intended (as per [comment](https://github.com/ReactiveMongo/ReactiveMongo/blob/13a2c9dcb5481f3c8af53ec9df4d8d52f16ba29d/driver/src/main/scala/api/collections/genericcollection.scala#L368)), but correct me if I'm wrong.
I have observed that the (correct) behavior for `ordered = false` was not changed by this PR.